docs/installation/installation-using-docker-macos.md

```diff
@@ -7,7 +7,9 @@ If you want to contribute to the framework itself,
 you need to install the whole [shopsys/shopsys](https://github.com/shopsys/shopsys) monorepo.
 Take a look at the article about [Monorepo](../introduction/monorepo.md) for more information.
 
-This solution uses [*docker-sync*](http://docker-sync.io/) (for fast two-way synchronization of the application files between the host machine and Docker volume).
+This solution uses [*docker-sync*](http://docker-sync.io/) (for relatively fast two-way synchronization of the application files between the host machine and Docker volume).
+
+**Warning: Docker-sync might be a burden for intensive project development, especially when you switch between branches or even between projects often. In such a case, you should consider using [native installation](./native-installation.md).**
```

Comment:
please be more descriptive about the huge amount of files in shared volumes of virtualized docker. if the big project would have just small portion of files that are shared, intensive switching between branches should not be such problem

Reply:
:thinking: but how can a huge project have shared just a small portion of files?

> huge amount of files in shared volumes of virtualized docker.

I would say that this state is by default after installing project-base, am I right?

Reply:
However, I have tried to add the information there, is it better now?